components/service/glean/src/test/java/mozilla/components/service/glean/TestUtil.kt

```diff
@@ -118,6 +120,8 @@ internal fun resetGlean(
     config: Configuration = Configuration(),
     clearStores: Boolean = true
 ) {
+    TimingManager.getElapsedNanos = { SystemClock.elapsedRealtimeNanos() }
```

Comment:
Do we need this? Isn't this the same of TimingManager.kt:37?

Reply:
This resets the timer to something "real" after it may have been overridden by a test.

Reply:
This shouldn't be needed for *all the tests*, right? Maybe we should just be careful about this in the specific test files (see below)